src/test/styles.test.js

```diff
@@ -55,6 +55,10 @@ describe('with styles', () => {
         & + & {
           margin-bottom: 4px;
         }
+
+        & > & {
```

Comment:
After looking at the tests, do we have snapshots for these three combinators in particular?

Reply:
Other than the ones in that group?

Reply:
sorry, my bad. I meant stylis level tests but I think we can live without them